README.md

```diff
@@ -41,6 +41,8 @@ git clone https://github.com/tribofustack/orderly.git
 
```

Comment:
IMO, seeing this without the knowledge of the project seems like we need to run these 2 commands to run the project. Also, the command is `start:docker` instead of `docker:start`.
I believe you should add only:
 
 ```sh
npm run start:docker
```

I don't think there's a need to add any more commands other than this one.

I would include also:

```markdown
This will launch the application and all dependencies needed. Once it's up, the [Swagger](https://swagger.io/) documentation can be accessed at http://localhost:3000/
```